packages/@ama-sdk/schematics/schematics/typescript/shell/templates/base/openapitools.json.template

```diff
@@ -8,7 +8,7 @@
       "<%=projectName%>-<%=projectPackageName%>": {
         "generatorName": "typescriptFetch",
         "output": ".",
-        "inputSpec": "./swagger-spec.yaml"
+        "inputSpec": ""
```

Comment:
Is it valid?

Reply:
Yes it is according to the schema.
It can be empty or `undefined`, I let it undefined to avoid having to find the field if a user wants to write it by hand.